packages/components/pie-card/src/defs.ts

```diff
@@ -8,7 +8,7 @@ export const paddingValues = spacingTokens.flatMap((first) => [first, ...spacing
 
 type PaddingValue = typeof spacingTokens[number];
 
-export type AriaProps = {
```

Comment:
We don't export the `AriaProps` types to avoid name collisions across components